### DISCUSSION TITLE
Welcome to Psychosecurity Research!

### DISCUSSION THREAD

**Discussion by HyperCrowd** [PsySecGroup/foundation#2] [General]
To get this community started, let's start with reading the [Guidance Document](https://github.com/PsySecGroup/foundation/blob/main/GUIDANCE.md), asking for clarifications, looking for improvements, and also see if we can find examples of psychosecurity techniques in the wild.

**Comment by wwwportal** [comment]
What happened to the psychosecurity program?

**Comment by HyperCrowd** [comment]
A lot of preliminary work is being done.  Unfortunately, the AI craze has made investors of social media that "_data is the new oil_" and so, they've all embraced anti-scraping practices.  At the time of this post, Reddit stock was worth $46.  Today, it is worth $133, off a peak of $225.

This has hampered a lot of core technology this project was using and we have been adapting ever since.
